packages/nextjs/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "@sentry/nextjs",
+  "version": "6.2.5",
+  "description": "Official Sentry SDK for Next.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/nextjs",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=6"
+  },
+  "module": "./esm/index.server.js",
+  "browser": "./esm/index.client.js",
+  "types": "./esm/react.d.ts",
```

```suggestion
  "types": "./esm/index.client.d.ts",
```

This got overlooked in the renaming.